doc/rtd/index.rst

```diff
@@ -77,7 +77,8 @@ projects, contributions, suggestions, fixes and constructive feedback.
 * Read our `Code of Conduct`_
 * Ask questions in the ``#cloud-init`` `room on Matrix <Matrix_>`_
 * Follow announcements or ask a question on `GitHub Discussions`_
-* :ref:`Contribute on GitHub<contributing>`
+* `Contribute on GitHub <GH repo_>`
+* :ref:`Contribution Guidelines <contributing>`
```

Comment:
Thank you for the contributio here @canonical/automation-and-tooling 
 Two things for this fix. 
1. First, the rst format referenced here doesn't link to anything. You can see the full generated content on a linux system by running `tox -e doc` then `xdg-open doc/rtd_html/index.html`. That rendered content has a broken undefined link for Contribute on GitHub. It should link to our repo at  https://github.com/canonical/cloud-init.
2. We should also adjust the link in doc/rtd/development/index.rst to make the text `Cloud-init is hosted and managed on GitHub` also point to the defined `.. _repository` link which is currently unused within the page.